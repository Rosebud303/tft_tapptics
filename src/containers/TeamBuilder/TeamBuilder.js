import React from 'react'

export default function TeamBuilder() {
  return (
    <div>
      <h1>build yo team here, kiddo</h1>
    </div>
  )
}
