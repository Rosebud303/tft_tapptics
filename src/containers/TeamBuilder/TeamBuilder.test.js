import React from 'react';
import { shallow } from 'enzyme';
import TeamBuilder from './TeamBuilder';

describe('TeamBuilder', () => {

  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<TeamBuilder/>);
  });

  it('should match the snapshot with all the data passed in', () => {
    expect(wrapper).toMatchSnapshot();
  });
});