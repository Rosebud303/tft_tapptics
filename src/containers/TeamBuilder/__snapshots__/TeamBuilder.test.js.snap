// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TeamBuilder should match the snapshot with all the data passed in 1`] = `
<div>
  <h1>
    build yo team here, kiddo
  </h1>
</div>
`;
